schemas/eng-workflow/build/build.1.schema.json

```diff
@@ -64,6 +64,7 @@
     },
     "client_id": {
       "description": "A UUID to uniquely identify a client",
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
```

Comment:
r? @cgsheeh as the original author of this field

Reply:
This field is generated using Python's `uuid.uuid4()` function. Testing with `re.match('^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$', str(uuid.uuid4()))` came up with a match, so I think this should be fine. r+